pkg/cli/environment.go

```diff
@@ -120,6 +125,18 @@ func envIntOr(name string, def int) int {
 	return ret
 }
 
+func envBoolOr(name string, def bool) bool {
```

Comment:
Please write tests for this new function.

Reply:
Done:)